include/tack_dummy.cpp

```diff
@@ -0,0 +1,3 @@
+// this file only exists to allow cmake to generate a valid library
+
+#include "tack/float4.h"
```

Comment:
I strongly prefer angle brackets for includes, only using quotes for relative paths.